modules/local/quast_assembly.nf

```diff
@@ -12,14 +12,16 @@ process QUAST {
 
     output:
     tuple val(meta), path("${prefix}/**"), path(contigs), emit: quast_data
-    tuple val(meta), path("${prefix}/${params.quast.report_prefix}${prefix}${params.quast.report_ext}"), path(contigs), emit: quast_table
+    tuple val(meta), path("${prefix}/quast_table/${params.quast.report_prefix}${prefix}${params.quast.report_ext}"), path(contigs), emit: quast_table
```

Comment:
This change was made because the quast transposed table (e.g., `short.transposed_short.quast.quality.tsv`) is matched by both this output (`quast_table`) and the `quast_data` output (which matches `**`). This meant that the nf-iridanext plugin was outputting two entries for the file `short.transposed_short.quast.quality.tsv`, which could not be differentiated from each other.

This change preserves the output of the file from this process in Nextflow named as `quast_table` by making a copy of this transposed table file under a different directory so that there is a way to differentiate the paths in the output. That is to say, now the output of this process will include for example:

*  `short/short.transposed_short.quast.quality.tsv`: This is what is included in the IRIDA Next JSON output.
* `short/quast_table/short.transposed_short.quast.quality.tsv`: This is what's emitted in this process as `quast_table`

Reply:
Thanks so much for handling this! 😄